iOS-sample-app/Showcases/Transactions/TransactionsViewController.swift

```diff
@@ -0,0 +1,49 @@
+import UIKit
+import ContentSquare
+
+struct Item
+{
+    private(set) var title: String
+    private(set) var price: Float
+    private(set) var currency: Currency
+
+    init(title: String, price: Float, currency: Currency)
+    {
+        self.title = title
+        self.price = price
+        self.currency = currency
+    }
+}
+
+// This class showcases how to track purchases made by an user with CustomerTransactions
+// Add some items to the cart, then validate it to send the transactions
+class TransactionsViewController: UIViewController {
+
+    private var cart = [Item]()
+    private var selectedCurrency : Currency = .eur
+    private var itemCount = 0
```

Comment:
You might want to start at 1, or use something different for the item price (could just be a static number like 5 for all items), or else if you only add one item you get a price of 0 - maybe you want that behaviour, but it might be somehow misleading).

Reply:
I agree with @CyberDandy that a static price might be less confusing

Reply:
Ok let's go for a static price